packages/workbox-webpack-plugin/src/lib/utils/copy-workbox-sw.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path');
+const {readFile} = require('./read-file');
+const errors = require('workbox-build/src/lib/errors');
+
+module.exports = (readFileFn) => {
```

Comment:
FWIW, in the v3 branch, we're transpiling all the Node code, so feel free to use features like `async`/`await` if you'd prefer.

Reply:
YAY!